### PR TITLE
chore: remove old event stream from codebase

### PR DESCRIPTION
**Pull request by mkanoor**
https://issues.redhat.com/browse/AAP-27632
Remove the old EventStream from fanout

### CODE REVIEW

**Comment by Dostonbek1** [comment]
@mkanoor Do you have a PR in plan for updating the e2e tests for these changes?

File: src/aap_eda/services/activation/status_manager.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ class StatusManager:
 
     def __init__(
         self,
-        db_instance: tp.Union[models.Activation, models.EventStream],
+        db_instance: models.Activation,
```
Comment (hsong-rh) [inline src/aap_eda/services/activation/status_manager.py:65]: The `EVENT_STREAM` had been removed from the `ProcessParentType` enum, we should remove the check on https://github.com/ansible/eda-server/pull/1008/files#diff-0140165e8f538211fec9cbc973dca39efb2e8c248cff89b4867cfa7c174d189cR73

File: src/aap_eda/core/utils/strings.py [hunk 2]
```diff
@@ -60,18 +60,6 @@ def substitute_variables(
...
-        "event_stream": event_stream,
-    }
-    for key in extra_vars:
-        context[key] = extra_vars[key]
-
-    source["args"] = substitute_variables(source.get("args", {}), context)
-    return source
-
-
 def substitute_extra_vars(
```
[the remark this answers is not in the document]
Comment (mkanoor) [reply src/aap_eda/core/utils/strings.py:63]: @hsong-rh We might need these functions as we add other parameters to event_streams parameters like certificates for PG Notify connection.

**Review by hsong-rh — DISMISSED** [review]

**Comment by mkanoor** [comment]
@Dostonbek1 I have generated a new API client in this PR https://github.com/ansible/eda-qa/pull/420

**Comment by mkanoor** [comment]
@Dostonbek1 e2e tests are passing with https://github.com/ansible/eda-qa/pull/420

**Comment by Dostonbek1** [comment]
@mkanoor sanity check for e2e run: https://github.com/ansible/eda-server/actions/runs/10428558625

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by hsong-rh — APPROVED** [review]

**Comment by rooftopcellist** [comment]
Closing this PR as the changes have landed as a part of https://github.com/ansible/eda-server/pull/1011.

**Pull request closed without merging**